junit/src/main/java/org/kaazing/k3po/junit/rules/SpecificationStatement.java

```diff
@@ -68,39 +68,43 @@ public void evaluate() throws Throwable {
             try {
                 // note: JUnit timeout will trigger an exception
                 statement.evaluate();
-            } catch (AssumptionViolatedException e) {
-
+            } catch (Throwable cause) {
```

Comment:
Can we switch this to catch `Exception` instead?
Catching `Throwable` is generally frowned upon because it catches internal JVM `Error` as well as `Exception`.